### DISCUSSION TITLE
Add iPhone camera photos to tracks

### DISCUSSION THREAD

**Discussion by n7ihq** [osmandapp/OsmAnd#20881] [Maps Related]
Is there a way to attach iPhone camera photos to recorded tracks? I want to use local saved iPhone photos, not Mapillary. Can I use an OpenStreetMaps editor to import photos and display on the OsmAnd iPhone app?

**Comment by ZeljkoBG** [comment]
He can't.
Mapillary is very bad service.

**Comment by dmpr0** [comment]
Hi! Currently, there is no way to attach images to tracks of favorites, but we plan to add it to favorites first.

**Reply by ZeljkoBG** [reply]
Are we so close, a good idea, that we will be able to add icons for our own POIs?

**Comment by n7ihq** [comment]
A workaround for adding photos to a track: Create a Waypoint and add a iCloud photo link to the Waypoint Description.

**Comment by n7ihq** [comment]
"2024 New Year Resolutions: A possibility to attach media from your library to favourites, tracks, waypoints."
Will this feature be implemented in 2024 for the iPhone?

**Reply by vshcherb** [reply]
Unfortunately, I think not much time left. Moved to next year

**Reply by n7ihq** [reply]
Is this feature request on the roadmap for 2026?
